server/routes/reviewPlan/exemptionReasonController.test.ts

```diff
@@ -28,19 +30,26 @@ describe('exemptionReasonController', () => {
   })
 
   describe('getExemptionReasonView', () => {
-    it(`should get 'exemption reason' view given form is already on the prisoner context`, async () => {
+    it(`should get 'exemption reason' view given dto is already on the prisoner context`, async () => {
       // Given
-      const expectedExemptionReasonForm = {
+      const reviewExemptionDto: ReviewExemptionDto = {
+        exemptionReason: 'EXEMPT_PRISONER_DRUG_OR_ALCOHOL_DEPENDENCY',
+        exemptionReasonDetails: {
```

Comment:
These tests are perfect 👍 , the only thing you'll need to change is this `exemptionReasonDetails` property on lines 37, 76 and 106; to reflect the fact that its a simple string:
```
      const expectedForm: ReviewExemptionForm = {
        exemptionReason: 'EXEMPT_PRISONER_DRUG_OR_ALCOHOL_DEPENDENCY',
        exemptionReasonDetails: 'In treatment',
      }
```